### PR TITLE
Vampire ability cost buff

### PR DESCRIPTION
**Pull request by kilkun**
Removed cost of evasion and stun oriented vampire powers. No longer shall creatures of the night be foiled by a librarian with a bar of soap!

Hopefully now Vampire rounds won't be extended 2.0 electric boogaloo.

Changes:

-Blood mist, Chiroptean screech, Hypnotise, Shapeshift all are now free to use once unlocked.

-Diseased touch cost reduced to 50u blood, for balance reasons.

-Shadowstep reduced to costing 10u blood

-Removed cost amount from verb names.

### CODE REVIEW

**Comment by IratePirate** [comment]
Good man.

While you're at it can you make the Ling horror form stronger?

**Comment by ComicIronic** [comment]
I'll leave this open for a discussion, but if nobody objects I'll mark it as ready.

**Comment by VelardAmakar** [comment]
After this goes through I think I'll work on some vampire additions that I should have worked on ages ago. After discussing them with you, that is.

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -124,12 +125,12 @@
 
 /client/proc/vampire_disease()
 	set category = "Vampire"
-	set name = "Diseased Touch (100)"
+	set name = "Diseased Touch"
```
Comment (Duny-) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Where can the user see the cost if not here? And why would you make it less expensive?

**Comment by clusterfack** [comment]
Can you for the sake of discussion put ALL your balance changes into your post?

**Comment by Duny-** [comment]
Diseased Touch should stay expensive; I also think Chiroptean Screech is too powerful to not have a price.

**Comment by kilkun** [comment]
Diseased Touch (much like Deathsting) has a reduced price because it's easily foiled (spaceacillin at the first sign of symptoms and it's gg) and it's slow acting. In order for 100 to even be close to a fair amount of used blood it would either have to be faster killing or not so easy to neutralize.

Shadowstep has a cost instead of blood mist because it's faster acting, goes hand in hand with cloak of darkness, and gives absolutely no notification to anybody. It also allows you to move while stunned. It kind of would be bullshit if an unrobust vampire kept shadow stepping away everytime he got tased. It seems like blood mist is stronger, as it allows you to go further and choose where you land, but shadow step is much MUCH stronger.

**Comment by Duny-** [comment]
Alright then. I thought Diseased Touch wasn't shit, my bad. I guess Shadowstep and Mist Form also make sense.  We'll see how it goes.

**Pull request merged**